native/SampleApps/SmartSyncExplorer/SmartSyncExplorerUITests/OAuthUITests.swift

```diff
@@ -109,7 +109,7 @@ class OAuthUITest: SalesforceNoSessionTestCase {
             sleep(1)
         }
         
-        SFAuthenticationManager.shared().logoutAllUsers() //FIXME: for some reason, this doesn's work
+        SFUserAccountManager.sharedInstance().logoutAllUsers() //FIXME: for some reason, this doesn's work
```

Comment:
What that FIXME about?

Reply:
@wmathurin  IMHO the future of these UItests are questionable. We don't use them. I only moved SFAuthenticationManager to SFUserAccountManager. The "Fixme" reason was not clear in the first place. I left it the way it was. We need to talk about the status of UI tests in the SDK. They aren't being used for testing.  Should we continue to keep them?

Reply:
(Outside this PR of course) we should have unit tests for a sample apps - especially for things that cannot be completely tested in unit tests e.g logout / login.